<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_vuelo</class>
 <widget class="QDialog" name="ventana_vuelo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vuelo</string>
  </property>
  <widget class="QLabel" name="lbl_escoger">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Escoja la clase de vuelo:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_first">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primera clase.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_business">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clase Business.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_economic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clase Econ√≥mica.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_motrar_seleccion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
